plugins/islandora_context_reaction_restrict_by_ip.inc

```diff
@@ -111,7 +111,7 @@ class IslandoraContextReactionRestrictByIp extends context_reaction {
       if (!strlen($range)) {
         continue;
       }
-      list($low, $high) = explode(':', $range);
+      list($low, $high) = array_pad(explode(':', $range, 2), 2, NULL);
```

Comment:
@MarcusBarnes the addition of `array_pad()` does get rid of the notice when given a single IP address without a `:`, but `NULL` is being assigned to `$high`, which is then being passed into `ip2long()`. I think we need to add logic to test whether `$high` is `NULL` and if so, determine if access from `$low` as a single IP address, not as the start of a range, is allowed. We'd also need to update the help text in the IP range field to indicate that a single IP is allowed.

If you're OK with the additional logic and help text, I can add it, or if you want to, go ahead.